pkg/kube/wait.go

```diff
@@ -219,7 +217,7 @@ func (c *Client) servicesReady(svc []v1.Service) bool {
 		}
 
 		// Make sure the service is not explicitly set to "None" before checking the IP
-		if s.Spec.ClusterIP != v1.ClusterIPNone && !helper.IsServiceIPSet(&s) {
+		if s.Spec.ClusterIP == v1.ClusterIPNone || s.Spec.ClusterIP == "" {
```

Comment:
This should still be `s.Spec.ClusterIP != v1.ClusterIPNone` because if the service is set to be `None`, then we can't check if it is ready

Reply:
I think `s.Spec.ClusterIP != v1.ClusterIPNone && s.Spec.ClusterIP == ""` should work

Reply:
@thomastaylor312 The above boolean expression had redundancies. I feel the `s.Spec.ClusterIP != v1.ClusterIPNone` part should be clubbed with the previous condition `if s.Spec.Type == v1.ServiceTypeExternalName {` rather than this.